src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/root/main.js

```diff
@@ -97,22 +104,33 @@ define(['sulumedia/model/media'], function(Media) {
         },
 
         bindCustomEvents: function() {
+            /**
+             * Change current view of the datagrid to given viewKey
+             * viewKey must be specified in listViews-Object
+             */
+            var changeDatagridView = function(viewKey) {
```

Comment:
@nnatter can we implement this as a standard object-method instead of making it private?